src/components/views/elements/PersistentApp.tsx

```diff
@@ -71,6 +83,17 @@ export default class PersistentApp extends React.Component<IProps, IState> {
         });
     };
 
+    private onWidgetAction = (payload: ActionPayload): void => {
+        switch (payload.action) {
+            case Action.AfterRightPanelPhaseChange:
+                this.setState({
+                    rightPanelPhase: RightPanelStore.getSharedInstance().roomPanelPhase,
+                });
+                break;
+            default: break;
```

Comment:
I do have a feeling this is unnecessary?

Reply:
This is fixed in the [follow up PR](https://github.com/matrix-org/matrix-react-sdk/pull/7450). ( to save me the work of debasing with merge conflicts, I hope that is fine.)